Vwitter/UIViewController+ErrorAlertPresenter.m

```diff
@@ -0,0 +1,24 @@
+//
+//  UIViewController+ErrorAlertPresenter.m
+//  Vwitter
+//
+//  Created by Christina Li on 7/11/22.
+//
+
+#import "UIViewController+ErrorAlertPresenter.h"
+
+@implementation UIViewController (ErrorAlertPresenter)
+
+- (void)presentErrorMessageWithTitle:(NSString *)title message:(NSString *)message {
+    UIAlertController* alert = [UIAlertController alertControllerWithTitle:title
+                               message:message
+                               preferredStyle:UIAlertControllerStyleAlert];
+
+    UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" style:UIAlertActionStyleDefault
```

Comment:
syntax nit: I would break each parameter into its own line like you did above

```
UIAlertAction* defaultAction = [UIAlertAction actionWithTitle:@"OK" 
                                   style:UIAlertActionStyleDefault
                                   handler:^(UIAlertAction * action) {}];
```